src/config/config-beta.ts

```diff
@@ -22,5 +24,13 @@ export const CONFIG_BETA: IAppConfig = {
         sendResponseUrl: extSyncBaseUrl + '/sendResponse',
         deleteRequestUrl: extSyncBaseUrl + '/deleteRequest'
     },
-    firebaseWebConfig: firebaseConfig
+    firebaseWebConfig: firebaseConfig,
+    notificationCenter: {
```

Comment:
i know we already used this pattern, but it's not really scalable, since we need to add all our endpoints in this config. So, a better approach would be to have only the `walletApiBaseUrl` in this config, the rest could stay in a specific client